wqflask/tests/wqflask/show_trait/test_export_trait_data.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+from wqflask.show_trait.export_trait_data import dict_to_sorted_list,cmp_samples
+class TestDictionaryList(unittest.TestCase):
```

Comment:
Best to have all the tests of one module/ class in one class; more-so when they are related. In this case, I'd put all the tests in a "TestExportTraits" class. The general rule of thumb(imho) is to have related methods in one class.